source/common/config/grpc_stream.h

```diff
@@ -71,7 +71,11 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
 
   bool grpcStreamAvailable() const { return stream_ != nullptr; }
 
-  void sendMessage(const RequestProto& request) { stream_->sendMessage(request, false); }
+  void sendMessage(const RequestProto& request) {
+    if (stream_ != nullptr) {
```

Comment:
Tests to cover this change.

Reply:
I am not sure this nullptr branch makes sense.
I feel it should be ASSERT for now. The crash is not deference nullptr, instead, the crash occurs because stream_ is a dangling pointer

Reply:
change to assert with better log